app/locale/PT/audit.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
There seem to be some formatting issues here, please use the project's formatting configuration to fix these (editorconfig, eslint)

Reply:
OK, I will fix in the next commit